auth/src/test/java/io/grpc/auth/GoogleAuthLibraryCallCredentialsTest.java

```diff
@@ -394,7 +394,7 @@ public void jwtAccessCredentialsInRequestMetadata() throws Exception {
     Map<?, ?> header = (Map<?, ?>) JsonParser.parse(jsonHeader);
     assertEquals("test-private-key-id", header.get("kid"));
     Map<?, ?> payload = (Map<?, ?>) JsonParser.parse(jsonPayload);
-    assertEquals("https://example.com:123/a.service", payload.get("aud"));
+    assertEquals("https://example.com/", payload.get("aud"));
```

Comment:
This will require an atomic upgrade of the auth library inside Google as well. We don't want to require that. Change the logic here to allow either the old or the new audience, with a comment.